changelog/13200.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+http:Fixing ResponseWriter to assert to http.Flusher in /sys/monitor endpoint
```

Comment:
Think about it from the user's perspective who's reading the CL.   Most of the above is meaningless to them.  Can you rewrite in a way that a user could understand it, without reference to internals?

Reply:
Sounds good! How about the following?
"Fixing `/sys/monitor` endpoint returning `streaming not supported`"

Reply:
Sounds good, though I suggest "Fix" rather than "Fixing", just to be consistent with how we usually phrase the CLs.